pkgs/applications/accessibility/espeakup/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenv, meson, ninja, espeak-ng, fetchFromGitHub, pkg-config, cmake, alsaLib, systemd }:
+
+stdenv.mkDerivation rec {
+  pname = "espeakup";
+  version = "0.90";
+  src = fetchFromGitHub {
+    owner = "linux-speakup";
+    repo = "espeakup";
+    rev = "569e9f5b0edbf427c5b877912ac4f016efd6a492";
+    sha256 = "sha256:0lmjwafvfxy07zn18v3dzjwwpnid2xffgvy2dzlwkbns8gb60ds2";
+  };
+
+  PKG_CONFIG_SYSTEMD_SYSTEMDSYSTEMUNITDIR = "${placeholder "out"}/lib/systemd/system";
+
+  nativeBuildInputs = [ meson ninja pkg-config cmake alsaLib systemd ];
```

Comment:
`alsaLib` is an alias of `alsa-lib` so this will not build with `nixpkgs.config.allowAliases = false;`.